app/src/main/java/uk/org/openseizuredetector/SdData.java

```diff
@@ -76,10 +79,26 @@ public class SdData implements Parcelable {
     private JSONObject jo;
     private JSONObject jsonObj;
     private JSONArray specArr;
+    public List<Double> heartRates = new ArrayList<Double>();
+    public void addNewHeartRateValue (double newHeartRateValue) {
```

Comment:
It looks like this will just keep adding values onto the list - how do we manage the length of the list?
I have added a CircBuffer (or something like that) class that is a fixed length of list - it over-writes old data when it is full - would that be better to avoid the list just getting longer all the time?